services/callback/src/main/resources/application.yaml

```diff
@@ -53,7 +55,7 @@ management:
     health:
       group:
         readiness:
-          include: db, verificationService
+          include: db, callbackController
```

Comment:
:astonished: `verificationService` ???

Reply:
As far as I understand this "callbackController" should correspond to a @Component("callbackController) which should inherit from HealthIndicator, which we do not have or? If yes then this can be removed